pages/awards/[id].js

```diff
@@ -367,12 +367,16 @@ const IndividualFeedback = ({ feedback }) => {
       </div>
       <h2>NOTES</h2>
       <p className={styles.individual__feedback_notes}>{feedback.notes || notes}</p>
-      <div className={styles.judge__feedback_audio_player}>
-        <p>{`OSSDF2021_1.mp3`}</p>
-        <span>
-          <p>{`3:07`}</p>
-          <img src={PlayIcon} />
-        </span>
+      <div className={styles.judge__feedback_audio_player_wrapper}>
+        <FeedbackAudio
+          audioUrl={feedback.audio_url}
+          edit={false}
+          setRecordingChanged={() => {}}
+          recording={null}
+          setRecording={() => {}}
+          recordedBlob={null}
+          setRecordedBlob={() => {}}
+        />
```

Comment:
If we're only allowing playback here, please use the `AudioPlayer` component.

Reply:
Good catch, will update!